### PR TITLE
Dvictor1/centos 6.5 mesos auth

### PR DESCRIPTION
**Pull request by dvictor1**

### CODE REVIEW

File: roles/chronos-sync/handlers/main.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+- name: Restart Chronos service
```
Comment (jonas) [inline roles/chronos-sync/handlers/main.yml]: I think this should go in the mesos-master role which installs chronis

File: roles/chronos-sync/tasks/common.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+- name: Install AWS CLI
+  pip: name=awscli state=present
+
+- name: Setup chronos sync
+  git: repo=https://github.com/mesos/chronos.git dest=/opt/chronos
+
+- name: Configure chronos password
```
Comment (jonas) [inline roles/chronos-sync/tasks/common.yml]: Also belongs in mesos-master role with everything below

File: roles/docker-engine/tasks/RedHat.yml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+---
+- name: repo install
+  template: src=RedHat_docker.j2 dest=/etc/yum.repos.d/docker.repo mode=0644
+
+- name: Install os packages
+  yum: name=docker-engine state=installed
+  
+- name: Start docker
```
Comment (jonas) [inline roles/docker-engine/tasks/RedHat.yml]: This could probably go into the common tasks

File: roles/chronos-sync/handlers/main.yml (hunk 1 above)
Comment (dvictor1) [inline roles/chronos-sync/handlers/main.yml]: chronos-sync moved

File: roles/docker-engine/tasks/RedHat.yml (hunk 3 above)
Comment (dvictor1) [inline roles/docker-engine/tasks/RedHat.yml]: docker-engine -> common

**Pull request closed without merging**